model/util.go

```diff
@@ -93,6 +93,14 @@ func unmarshalString(data []byte) (string, error) {
 	return value, nil
 }
 
+func unmarshalBool(data []byte) (bool, error) {
```

Comment:
See my other comment. No need to return an error here since you're not using it anyway. We can use the convention `must` and add a note in the func doc.